src/pool/srv_util.c

```diff
@@ -637,6 +637,106 @@ int ds_pool_get_failed_tgt_idx(const uuid_t pool_uuid, int **failed_tgts,
 	return rc;
 }
 
+/*
+ * Get and exclude all active target indexes on the current node, except for
```

Comment:
I don't quite get this, if a target isn't active (in up or upin) and not in specified target list, will it be excluded or not?